kolibri/core/assets/src/vue/textbox.vue

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
Apparently I missed this in my initial review - but our convention hitherto has been to have components as `index.vue` under the folder name, so this should be `textbox/index.vue`.